caffeine@patapon.info/extension.js

```diff
@@ -194,17 +198,22 @@ class CaffeineToggle extends QuickSettings.QuickMenuToggle {
     _syncTimers() {
         this._itemsSection.removeAll();
         this._timerItems.clear();
+        const durationIndex = this._settings.get_int(DURATION_TIMER_INDEX);
 
         for (const timer of TIMERS) {
-            const key = Object.keys(timer);
-            const label = timer[key][0];
+            let label = null;
+            if(timer[0] === 0) {
+                label = _('Infinite');
+            } else {
+                label = timer[durationIndex] + ":00";
```

Comment:
```suggestion
                label = parseInt(timer[durationIndex]) + 'm';
```

Maybe something like `5m`, `10m`... is more readable

Also I'd prefer to properly convert ints to strings before concatenation